### PR TITLE
clean up inputs tests for faster runs- functional and styles

### PR DESCRIPTION
**Pull request by umahaea**
clean up inputs tests for faster runs - functional and styles

Before: 647 tests
After: 241 tests

### CODE REVIEW

File: src/test/java/elementsSDKTests/functionalTests/InputsTest.java [hunk 1]
```diff
@@ -44,11 +44,18 @@
...
-    List<String> borderColors = Arrays.asList("border-top-color", "border-right-color", "border-bottom-color", "border-left-color");
-    List<String> borderRadii = Arrays.asList("border-top-left-radius", "border-top-right-radius", "border-bottom-right-radius", "border-bottom-left-radius");
-    List<String> paddings = Arrays.asList("padding-top", "padding-bottom", "padding-right", "padding-left");
+
+    String[] borderTops = {"border-top-width", "border-top-style", "border-top-color"};
+    String[] borderBottoms = {"border-bottom-width", "border-bottom-style", "border-bottom-color"};
+    String[] borderRights = {"border-right-width", "border-right-style", "border-right-color"};
+    String[] borderLefts = {"border-left-width", "border-left-style", "border-left-color"};
+    String[] paddings = {"padding-top", "padding-bottom", "padding-right", "padding-left"};
+    String[] borderRadiuss = {"border-top-left-radius", "border-top-right-radius", "border-bottom-right-radius", "border-bottom-left-radius"};
```
Comment (wyseguyonline) [inline src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: Is this line redundant with 58?

Comment (udhadpa) [reply src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: good catch. same with elements test

File: src/test/java/elementsSDKTests/functionalTests/InputsTest.java [hunk 2]
```diff
@@ -1402,132 +1403,67 @@ private void fancySelectInputFocusTest(String type, String className, String id,
...
+            log.info("Line Height for " + type + " is not as per the spec, actual: " + lineHeight);
+        }
+        if (!isColor) {
+            log.info("color for " + type + " is not as per the spec, actual: " + color);
+        }
+        if (!isBackgroundColor) {
+            log.info("bg-color for " + type + " is not as per the spec, actual: " + backgroundColor);
         }
-        Assert.assertTrue(isCSSProperty);
+        Assert.assertTrue(isHeight && isPaddingLeft && isPaddingRight && isMarginTop && isFontSize && isLineHeight && isColor);
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: missing assertion for isBackgroundColor

File: src/test/java/elementsSDKTests/functionalTests/InputsTest.java [hunk 3]
```diff
@@ -1994,67 +1946,174 @@ private void singleLineTextInputWiWoValuesMobileTest(String type, By element, St
...
+
+        fontSize = commonUtils.getCSSValue(element, "font-size", "mobile");
+        isFontSize = commonUtils.assertCSSProperties("font-size", fontSize, expFontSize);
+        if (!isFontSize) {
+            log.info("Compounds-> font-size for " + inputState + " is not as per the spec, actual: " + fontSize);
+        }
+        color = commonUtils.getCSSValue(element, "color", "mobile");
+        isColor = commonUtils.assertCSSProperties("color", color, expColor);
+        if (!isColor) {
+            log.info("Compounds-> color for " + inputState + " is not as per the spec, actual: " + color);
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: we no longer use Compounds. So should we remove this from log messages? Its a low priority though

**Comment by udhadpa** [comment]
Superb work Eajaz. I can imagine the hard work put into this. This was a long due and to see such a huge difference in tests is highly commendable. 💯

**Comment by umahaea** [comment]
many approved. so merging it.

**Comment by umahaea** [comment]
Thanks Payal for ur comments!

**Pull request merged**